index.js

```diff
@@ -187,6 +189,28 @@ var new_client = function(api_key, config) {
 
   }
 
+  client.all_flags = function(user, fn) {
+    var cb = fn || noop;
+    var _self = this;
+
+    eval_flags = function() {
+      cb(null, Object.keys(_self.features).reduce(function(previous, current) {
+        previous[current] = evaluate(_self.features[current], user);
+        return previous;
+      }, {}));
+    };   
+
+    if (!this.stream) {
+      cb(new Error("[LaunchDarkly] fetching all flags requires streaming mode enabled"));
+    }
+    else if (!this.initialized) {
+      this.initializeStream(eval_flags);
```

Comment:
It occurs to me that this will cancel any pending initialization attempt and re-start it, which is a bit inefficient. I think what I'd like to do is detect when initialization is pending and append this callback to a list of callbacks to be called when initialization completes.